### PR TITLE
add deprecation warning for docker_archive

### PR DESCRIPTION
**Pull request by MalteEbner**
## Description
Add a deprecation for all deprecation warnings in the Docker Archive

## Example screenshot

image

### CODE REVIEW

File: docs/source/docker_archive/advanced/active_learning.rst [hunk 1]
```diff
@@ -3,6 +3,12 @@
 Active Learning
 ===============
 
+.. warning::
+    **The Docker Archive documentation is deprecated**
+
+    The old workflow described in these docs will not be supported with new Lightly Worker versions above 2.6.
+    Please switch to our `new documentation page <https://docs.lightly.ai>`_ instead.
```
Comment (IgorSusmelj) [inline docs/source/docker_archive/advanced/active_learning.rst:10]: This looks great and should cover the most important pages! One small suggestion. Let's link to `https://docs.lightly.ai/docs` instead. Then they are directly on the right page.

**Review by IgorSusmelj — CHANGES_REQUESTED** [review]
Just one small suggestion. Otherwise this looks good :)

File: docs/source/docker_archive/advanced/active_learning.rst (hunk 1 above)
Comment (MalteEbner) [reply docs/source/docker_archive/advanced/active_learning.rst]: I changed it.

**Review by IgorSusmelj — APPROVED** [review]
LGTM

**Pull request merged**